hal/armv7m/imxrt/117x/imxrt117x.h

```diff
@@ -26,7 +26,7 @@ extern int hal_platformctl(void *ptr);
 extern unsigned int _imxrt_cpuid(void);
 
 
-extern void _imxrt_cleanDCache(void);
+extern void _imxrt_cleaninvalDCacheAddr(void *addr, u32 sz);
```

Comment:
Rename to `_imxrt_cleanInvalDCacheAddr`.

Reply:
Done